fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -508,6 +512,32 @@ private void updateMeta(Set<String> mvRefreshedPartitions,
         }
     }
 
+    private void updateAssociatedPartitionMeta(MaterializedView.AsyncRefreshContext refreshContext,
+                                               Set<String> mvRefreshedPartitions,
+                                               Map<Table, Set<String>> refTableAndPartitionNames) {
+        Map<String, Set<String>> mvToBaseNameRef = mvContext.getMvRefBaseTableIntersectedPartitions();
+        if (mvToBaseNameRef != null) {
+            try {
+                Table refBaseTable = refTableAndPartitionNames.keySet().iterator().next();
```

Comment:
what if it's empty

Reply:
`refTableAndPartitionNames ` should always not empty for partitioned mv and should only contain one element.

I'll add a check.